src/prog/main.F90

```diff
@@ -1571,6 +1581,12 @@ subroutine parseArguments(env, args, inputFile, paramFile, accuracy, lgrad, &
             call set_opt(env,'optlevel',sec)
          end if
 
+      case('--nat')
+         call args%nextArg(sec)
+         if (allocated(sec)) then
+            call set_natom(env,sec)
+         end if
```

Comment:
What is this command for?

Reply:
This is required for setting the number of atoms of the first molecule in the input structure of a xtb-IFF single point calculation.

Reply:
Why a fixed range (1-n)? We have much more flexible input models for fragments available.

Reply:
That's a good point. I will change this.